src/controller/c/messages.h

```diff
@@ -75,7 +75,7 @@
 #define C_SUPERVISOR_RELOAD_WORLD 52
 #define C_SUPERVISOR_SET_LABEL 53
 #define C_SUPERVISOR_SIMULATION_QUIT 54
-#define C_SUPERVISOR_SIMULATION_RESET 55
+// #define C_SUPERVISOR_SIMULATION_RESET 55  // Bidirectional as of R2025a
```

Comment:
```suggestion
```
I would simply remove this code as we generally don't want to accumulate commented out code.